--- origsrc/libofx-0.9.0/Makefile.am	2007-10-27 04:32:38.000000000 -0500
+++ src/libofx-0.9.0/Makefile.am	2007-11-29 23:41:17.466875000 -0600
@@ -3,8 +3,6 @@
 endif
 SUBDIRS = m4 inc dtd lib doc . ofx2qif ofxdump $(MAYBE_OFXCONNECT)
 
-docdir = ${prefix}/share/doc/libofx
-
 doc_DATA = \
   AUTHORS \
   COPYING \
--- origsrc/libofx-0.9.0/configure.in	2007-11-19 10:41:40.000000000 -0600
+++ src/libofx-0.9.0/configure.in	2007-11-30 07:12:48.154375000 -0600
@@ -283,11 +283,11 @@
 	AC_MSG_WARN([libcurl is not available. ofxconnect (Direct connect samples) will NOT be built.])
 fi
 
-PKG_CHECK_MODULES(LIBXMLPP,libxml++-1.0 >= 1.0.1,
+PKG_CHECK_MODULES(LIBXMLPP,libxml++-2.6,
 	[
 	AC_DEFINE(HAVE_LIBXMLPP, 1, [Defined if libxml++ is available])
 	have_libxmlpp=yes],
-        [AC_MSG_WARN([libxml++ 1.0 is not available. ofxconnect (Direct connect samples) will NOT be built.])
+        [AC_MSG_WARN([libxml++ 2.6 is not available. ofxconnect (Direct connect samples) will NOT be built.])
 	have_libxmlpp=no])
 
 #PKG_CHECK_MODULES(QT,qt-mt >= 3.2,
--- origsrc/libofx-0.9.0/doc/Makefile.am	2005-07-31 14:48:50.000000000 -0500
+++ src/libofx-0.9.0/doc/Makefile.am	2007-11-30 07:06:07.045000000 -0600
@@ -1,7 +1,5 @@
 SUBDIRS =
 
-docdir = ${prefix}/share/doc/libofx
-
 EXTRA_DIST = \
 	doxygen.cfg \
 	ofx_sample_files \
@@ -15,7 +13,7 @@
 html/: doxygen.cfg
 	echo "doc: " && pwd && 	echo "distdir: " && echo $(distdir)
 	rm -rf html/ refman.pdf
-	$(DOXYGEN) doxygen.cfg
+	$(DOXYGEN) $<
 #	$(MAKE) -C latex/ 
 #	mv latex/refman.pdf ./refman.pdf
 
--- origsrc/libofx-0.9.0/lib/ofx_preproc.cpp	2007-11-12 00:07:48.000000000 -0600
+++ src/libofx-0.9.0/lib/ofx_preproc.cpp	2007-11-29 23:24:01.904375000 -0600
@@ -185,7 +185,7 @@
 	    memset(iconv_buffer,0,READ_BUFFER_SIZE);
 	    size_t inbytesleft = strlen(s_buffer.c_str());
 	    size_t outbytesleft = READ_BUFFER_SIZE;
-#ifdef OS_WIN32
+#if defined(OS_WIN32) || defined(__CYGWIN__)
 	    const char * inchar = (const char *)s_buffer.c_str();
 #else
 	    char * inchar = (char *)s_buffer.c_str();
--- origsrc/libofx-0.9.0/libofx.pc	2007-11-19 13:25:04.000000000 -0600
+++ src/libofx-0.9.0/libofx.pc	2007-11-30 14:32:13.000000000 -0600
@@ -1,7 +1,7 @@
 # libofx pkg-config source file
 
 prefix=/usr
-exec_prefix=${prefix}
+exec_prefix=/usr
 libdir=${exec_prefix}/lib
 includedir=${prefix}/include
 
